ports/espressif/boards/m5stack_m5paper/pins.c

```diff
@@ -80,6 +80,5 @@ static const mp_rom_map_elem_t board_module_globals_table[] = {
     { MP_ROM_QSTR(MP_QSTR_UART), MP_ROM_PTR(&board_uart_obj) },
 
     // TODO: Uncomment once the display initialization code is added.
-    // { MP_ROM_QSTR(MP_QSTR_DISPLAY), MP_ROM_PTR(&displays[0].display)}
```

Comment:
Delete all of these.

Reply:
is there more than one?

Reply:
Sorry, I meant all of the selected lines. This is the only spot I saw.